app/dependencies.php

```diff
@@ -177,7 +175,7 @@
       VersionParser::class => autowire(VersionParser::class),
       Twig::class => function (ContainerInterface $container): Twig {
         $cache = false;
-        if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'production') {
+        if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'prod') {
```

Comment:
@okiedork should this be kept as "production"?

Reply:
Let's use `prod` here.  We are already using it like that.